support-frontend/assets/pages/supporter-plus-landing/supporterPlusRouter.tsx

```diff
@@ -47,6 +47,37 @@ const isInTwoStepTest =
 const showCheckoutTopUpToggle =
 	abParticipations.twoStepCheckoutWithNudgeBelow === 'variant_b';
 
+const checkCanMakePaymentsWithActiveCard =
+	abParticipations.canMakePaymentsWithActiveCard &&
+	abParticipations.canMakePaymentsWithActiveCard == 'control';
+
+/**
+ * Temporary check behind disabled AB test to validate whether
+ * window.ApplePaySession.canMakePaymentsWithActiveCard works
+ * on support.theguardian.com
+ */
+if (checkCanMakePaymentsWithActiveCard) {
+	const merchantIdentifier = 'merchant.uk.co.guardian.contributions';
+
+	const canMakePaymentsWithActiveCard = (): Promise<boolean> => {
+		return new Promise((resolve) => {
+			if (window.ApplePaySession) {
+				void window.ApplePaySession.canMakePaymentsWithActiveCard(
+					merchantIdentifier,
+				).then((result) => {
```

Comment:
Also seems to need the return type specified now, this is the correction suggested  `).then((result: boolean | PromiseLike<boolean>) => {`